src/ssh/azext_ssh/azext_metadata.json

```diff
@@ -1,4 +1,4 @@
 {
-    "azext.isPreview": false,
+    "azext.isPreview": true,
```

Comment:
Please notify sandeep that we are changing the preview with SSHArc changes. Given we have a unified experience this is expected. I'm not sure if there is something that shouldbe communicated / documented to customers as the previous version is marked as GA but the new version as preview.

Reply:
It's a bad practice to convert GA extension back to preview as this will cause unexpected issues. We have seen this before...